app/Http/Controllers/Auth/AuthController.php

```diff
@@ -52,7 +51,7 @@ public function handleTwitterCallback()
             Auth::login($user);
         } else {
             Auth::login($user = User::create([
-                'name' => $twitter->name,
+                'name'       => $twitter->name,
```

Comment:
guessing this is php storm, but this codebase (and all of mine because I'm picky and opinionated and annoying) = no indented alignment :)